### ISSUE TITLE
Jumping back to the home page after loggin in. 

### ISSUE THREAD

**Issue by MingCWang** [MingCWang/deiseval#91]
**Describe the bug**
After user logs in with OAuth, it always redirects the user back to the home page regardless where the user original were.

**To Reproduce**
1. Navigate to a page that is not the home page first
2. Login with your Brandeis credentials. 
3. You will see the redirection

**Expected behavior**
The users should return the original page they were on and NOT the home page.

**Comment by cpwallah** [comment]
hi i would kike to work on this issue @MingCWang

**Comment by MingCWang** [comment]
> hi i would kike to work on this issue @MingCWang

Of course! I will assign you to this issue. Just curious, how did you find this project?

**Comment by cpwallah** [comment]
good first issue thank to that website :)

**Comment by MingCWang** [comment]
> good first issue thank to that website :)

Npi! Did you search google with a keyword or did you find this randomly? I'm asking because I'm curious how this website would appear on google search.

**Comment by cpwallah** [comment]
i found randomly

**Comment by MingCWang** [comment]
> i found randomly

it might be a bit tricky to test in the prod environment since you don't have the Brandeis credentials, but it should be fine if it works in dev. I just added details for the `.env` files [here](https://github.com/MingCWang/deiseval/blob/main/CONTRIBUTING.md#env-file)
